TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/CapturedTrafficToHttpTransactionAccumulator.java

```diff
@@ -200,7 +200,8 @@ public CONNECTION_STATUS addObservationToAccumulation(String nodeId, Accumulatio
             rotateAccumulationIfNecessary(connectionId, accum);
             exceptionConnectionCounter.incrementAndGet();
             accum.resetForNextRequest();
-            log.atWarn().setMessage(()->"Removing accumulated traffic pair for " + connectionId).log();
+            log.atDebug().setMessage(()->"Removing accumulated traffic pair due to " +
```

Comment:
I'm curious that we apparently caught an exception here and it's still just a debug level log. Why so low?

Reply:
It was an exception on the socket in the capture.  It should be as recoverable to the user as it is with this process if wants to know more.  

My reasoning was this the replayer is behaving normally, so there's no reason to warn a user.  It could be a warning for the original client, or even the capture.